PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,13 @@
 # Description
 
 <!--
-Please include a summary of the change and which issue is fixed -include its number-. Please also include relevant motivation and context.
+Please include a summary of the change and which issue is fixed -include its number-. It's important that PRs connect to an existing issue, and we'll review this PR in part based on the content of that issue. Please also include relevant motivation and context.
 -->
 
 # Checklist:
 
 - [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
+  - [ ] It references and cleanly addresses an existing, filed issue
```

Comment:
```suggestion
  - [ ] This PR fixes Issue #<!-- replace with relevant issue -->
```

Reply:
I sort of like it, sort of don't.  Often a PR is related to, but does not completely address, an issue.  Git will auto close an issue if you say `fixes #x`... although I'm not sure that applies with the word issue in between.

Reply:
yes, perhaps better is

```suggestion
- [ ] This PR addresses Issue #<!-- replace with relevant issue -->
```

Reply:
I gave IDs their own sections. What do you think?

Reply:
I like it.  But now is the description change and checklist addition overkill?

Reply:
Fair! I removed the checklist item.